src/Microsoft.Health.Dicom.SqlServer/Microsoft.Health.Dicom.SqlServer.csproj

```diff
@@ -54,6 +58,11 @@
       <InputToMutableSqlGenerator>true</InputToMutableSqlGenerator>
       <MutableClassVersion>Latest</MutableClassVersion>
     </EmbeddedResource>
+    <EmbeddedResource Include="Features\Schema\Migrations\3.sql">
+      <InputToImmutableSqlGenerator>true</InputToImmutableSqlGenerator>
+      <InputToMutableSqlGenerator>true</InputToMutableSqlGenerator>
+      <MutableClassVersion>3</MutableClassVersion>
```

Comment:
Should _MutableClassVersion_ be _Latest_  for 3.sql , while the 2.sql become _2_?

Reply:
I think I will switch it to that in a bit. For now I'll add the schema separately and rename when I integrate code to call it!